### PR TITLE
docs(protocol): update mainnet deployment doc based on the latest batch transaction

### PR DESCRIPTION
**Pull request by davidtaikocha**

### CODE REVIEW

File: packages/protocol/deployments/mainnet-contract-logs.md [hunk 1]
```diff
@@ -28,6 +28,8 @@
   - bridged_erc1155: `0x3c90963cfba436400b0f9c46aa9224cb379c2c40`
   - bridged_erc721: `0xc3310905e2bc9cfb198695b75ef3e5b69c6a1bf7`
   - bridged_erc20: `0x79bc0aada00fcf6e7ab514bfeb093b5fae3653e3`
+  - bridge_watchdog: `0x00000291ab79c55dc4fcd97dfba4880df4b93624`
+  - quota_manager: `0x91f67118DD47d502B1f0C354D0611997B022f29Es`
```
Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs.md:32]: please also add quota manager related info below.

Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs.md:32]: remove the ending 's'?

Comment (dantaik) [reply packages/protocol/deployments/mainnet-contract-logs.md:32]: please also list all tokens and their quota in this file as well.

Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-contract-logs.md]: removed

File: packages/protocol/deployments/mainnet-contract-logs.md [hunk 2]
```diff
@@ -141,7 +142,7 @@
   - tier_guardian: `0xE3D777143Ea25A6E031d1e921F396750885f43aC`
   - automata_dcap_attestation: `0x8d7C954960a36a7596d7eA4945dDf891967ca8A3`
   - proposer_one: `0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045` vitalik.eth
-  -
+  - proposer: `0x000000633b68f5d8d3a86593ebb815b4663bcbe0`
```
Comment (dantaik) [inline packages/protocol/deployments/mainnet-contract-logs.md:145]: why this is a proposer here?

File: packages/protocol/deployments/mainnet-contract-logs.md (hunk 1 above)
Comment (davidtaikocha) [reply packages/protocol/deployments/mainnet-contract-logs.md]: Added: https://github.com/taikoxyz/taiko-mono/pull/17162/commits/8fbdf698ec07418ff6ccd911a3b5afb3801d4e3a

**Review by dantaik — APPROVED** [review]

**Pull request merged**